src/helpers/github.ts

```diff
@@ -33,7 +33,6 @@ export async function getMergeTimeoutAndApprovalRequiredCount(context: Context,
   };
   const timeoutContributor = {
     mergeTimeout: mergeTimeout?.contributor,
-    requiredApprovalCount: approvalsRequired?.contributor,
```

Comment:
I don't think that is proper. If the user is a contributor then no `requiredApprovalCount` will be returned, which subsequently is most likely to fail when calling `attemptMerging`. Could you provide some QA?

Reply:
Sure. I'm modifying the logic and creating QA tests.